packages/suite/src/components/suite/FiatValue.tsx

```diff
@@ -76,13 +76,13 @@ export const FiatValue = ({
 }: FiatValueProps) => {
     const { FiatAmountFormatter } = useFormatters();
     const localCurrency = useSelector(selectLocalCurrency);
-    const coins = useSelector(selectCoins);
+    const coins = useSelector(selectCoinsLegacy);
```

Comment:
Do we still need this legacy selector? Wouldn't be easier to use selectFiatRatesByFiatRateKey instead directly? Maybe not, I don't know.